assets/pages/digital-subscription-checkout/helpers/paymentProviders.js

```diff
@@ -37,8 +56,21 @@ function create(state: State, token: string) {
 }
 
 function showPaymentMethod(state: State) {
-  setupStripeCheckout((token: string) => create(state, token), null, 'GBP', false)
-    .then(() => openDialogBox(10, 'test@test.com'));
+  const { currencyId, countryGroupId } = state.common.internationalisation;
+  const { paymentMethod, isTestUser } = state.page.checkout;
+  const price = getDigitalPrice(countryGroupId, state.page.checkout.billingPeriod);
+  switch (paymentMethod) {
+    case 'Stripe':
+      loadStripe()
+        .then(() => setupStripeCheckout((authorisation: StripeAuthorisation) => create(state, authorisation.token), 'REGULAR', currencyId, isTestUser))
+        .then(stripe => openDialogBox(stripe, price.value, state.page.checkout.email));
+      break;
+    case 'DirectDebit':
+      console.log('Direct Debit');
```

Comment:
Should these log statements be removed before shipping?

Reply:
No this is all still work in progress, I've got to implement that next.

Reply:
thanks for explaining